reactive-banana/src/Reactive/Banana/Prim/Types.hs

```diff
@@ -23,7 +23,7 @@ import Reactive.Banana.Prim.Util
 -- | A 'Network' represents the state of a pulse/latch network,
 data Network = Network
     { nTime           :: !Time                 -- Current time.
-    , nOutputs        :: !(OrderedBag Output)  -- Remember outputs to prevent garbage collection.
+    , nOutputs        :: {-# unpack #-} !(OrderedBag Output)  -- Remember outputs to prevent garbage collection.
```

Comment:
Avoids actually allocating an `OrderedBag` when creating an updated `Network` after stepping - instead we can just store the underlying pointers that we have access to from some worker/wrapper transformations.